pkg/kubelet/dockertools/docker.go

```diff
@@ -211,6 +211,7 @@ func (p throttledDockerPuller) IsImagePresent(name string) (bool, error) {
 	return p.puller.IsImagePresent(name)
 }
 
+// TODO (random-liu) Almost never used, should we remove this?
```

Comment:
Ha, what I mean is that we defined the the type "DockerContainers", and defined the function "FindPodContainer". But it seems that we just use "DockerContainers" as slice of APIcontainers (https://github.com/kubernetes/kubernetes/search?utf8=%E2%9C%93&q=GetKubeletDockerContainers&type=Code), never use function like FindPodContainer. So I just wonder whether we should remove this type, :)